Euler/mathfonts.css

```diff
@@ -0,0 +1,32 @@
+/*
+Euler Math is released under the SIL Open Font License. See the files in this
+directory for details. The font can be obtained from several TeX distributions
+or package managers. It is recommended to use it with the Lato font or other
```

Comment:
I'm not sure if there is an recommended text font for Euler Math. Did you really write Lato font on purpose?

Reply:
The Euler math fonts were developed for Knuth's _Concrete Mathematics_ (see https://www.ctan.org/pkg/euler) which uses the Concrete text font, but it's also commonly used with Palatino, for example Ravi Vakil's [_The Rising Sea_](http://math.stanford.edu/~vakil/216blog/FOAGjul2724public.pdf)

Reply:
Both fonts I mention are shown under the Euler-Math section of this TeX-SX answer: https://tex.stackexchange.com/questions/425098/which-opentype-math-fonts-are-available

Reply:
Sorry, copy and paste error. I am just back from skiing, will update the PR and send more later on.

Reply:
@fred-wang I updated the text concerning text font